### PR TITLE
[materialx] Fix features build

### PR DESCRIPTION
**Pull request by michaelmigliore**
`${FEATURE_OPTIONS}` was overwritten by explicitly setting the options to `NO` just below.  
Basically no feature could be enabled before this fix.

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- ~~[ ] SHA512s are updated for each updated download.~~
- ~~[ ] The "supports" clause reflects platforms that may be fixed by this new version.~~
- ~~[ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- ~~[ ] Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/materialx/portfile.cmake [hunk 1]
```diff
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO AcademySoftwareFoundation/MaterialX
     REF "v${VERSION}"
-    SHA512 ca743e619f51bddd67419c79a31e9fb92dd7883e8c182897c1d8cea2e5dc51cddf13ac8cc798cfa0f022dacf4fd77881aefc24f3184f8b7273651ba55c7df400
+    SHA512 ca743e619f51bddd67419c79a31e9fb92dd7883e8c182897c1d8cea2e5dc51cddf13ac8cc798cfa0f022dacf4fd77881aefc24f3184f8b7273651ba55c7df400
```
Comment (MonicaLiu0311) [inline ports/materialx/portfile.cmake:5]: Please restore this line to avoid unnecessary `balme`.

**Comment by MonicaLiu0311** [comment]
All features are tested successfully in the following triplet:
```
x86-windows
x64-windows
x64-windows-static
```

File: ports/materialx/portfile.cmake (hunk 1 above)
Comment (dg0yt) [reply ports/materialx/portfile.cmake:5]: This change *restores* the consistent EOL. It doesn't need to be reverted. 

@jiayuehua Your scripted updates must not break EOL.

Comment (MonicaLiu0311) [reply ports/materialx/portfile.cmake:5]: OK, thanks.

**Review by MonicaLiu0311 — APPROVED** [review]

**Pull request merged**